Gemfile

```
@@ -12,6 +12,8 @@ else
   gem "decidim", DECIDIM_VERSION
 end
 
+gem "decidim-cdtb", git: "https://github.com/CodiTramuntana/decidim-module-cdtb", branch: "main"
```

```suggestion
gem "decidim-cdtb", git: "https://github.com/CodiTramuntana/decidim-module-cdtb"
```

can you try without branch? `main` is default branch and that change it should work

---

Without main it looks for "master" and gives error